yt/frontends/parthenon/io.py

```diff
@@ -0,0 +1,107 @@
+from itertools import groupby
+
+import numpy as np
+
+from yt.utilities.io_handler import BaseIOHandler
+from yt.utilities.logger import ytLogger as mylog
+
+
+# http://stackoverflow.com/questions/2361945/detecting-consecutive-integers-in-a-list
+def grid_sequences(grids):
+    g_iter = sorted(grids, key=lambda g: g.id)
+    for _, g in groupby(enumerate(g_iter), lambda i_x1: i_x1[0] - i_x1[1].id):
+        seq = [v[1] for v in g]
+        yield seq
+
+
+ii = [0, 1, 0, 1, 0, 1, 0, 1]
+jj = [0, 0, 1, 1, 0, 0, 1, 1]
+kk = [0, 0, 0, 0, 1, 1, 1, 1]
+
+
+class IOHandlerParthenon(BaseIOHandler):
+    _particle_reader = False
+    _dataset_type = "parthenon"
+
+    def __init__(self, ds):
+        super().__init__(ds)
+        self._handle = ds._handle
+
+    def _read_particles(
+        self, fields_to_read, type, args, grid_list, count_list, conv_factors
+    ):
+        pass
+
+    def _read_fluid_selection(self, chunks, selector, fields, size):
+        chunks = list(chunks)
+        if any((ftype != "parthenon" for ftype, fname in fields)):
+            raise NotImplementedError
```

Comment:
I'm not sure this is reachable, and I find it a bit confusing
```suggestion
```

Reply:
This was copied over from the Athena++ frontend.
Could it have been added if people forcibly overwrite the `dataset_type`?
I don't see any reason why this would be the case and now have removed those two lines.